content/_index.de.md

```diff
@@ -0,0 +1,116 @@
+
+**Organic Maps** ist heutzutage eine der wenigen Anwendungen, die 100% der Funktionen ohne aktive Internetverbindung unterstützt. Installieren Sie Organic Maps, laden Sie Karten herunter, werfen Sie Ihre SIM-Karte weg (Ihr Betreiber verfolgt Sie übrigens ständig) und machen Sie einen Wochenausflug mit einer einzigen Akkuladung, ohne dass ein Byte an das Netz gesendet wird.
+
+
+{{ badges(lang='de') }}
+
+{{ screenshot(src='/images/screenshots/hiking.jpg', alt='Hiking') }}
+{{ screenshot(src='/images/screenshots/prague.jpg', alt='Prague') }}
+{{ screenshot(src='/images/screenshots/search.jpg', alt='Offline Search') }}
+{{ screenshot(src='/images/screenshots/dark.jpg', alt='Navigation in dark mode') }}
+
+## Funktionen
+
+Organic Maps ist die ultimative Begleit-App für Reisende, Touristen, Wanderer und Radfahrer:
+
+- Detaillierte Offline-Karten mit Orten, die auf anderen Karten nicht zu finden sind, dank [OpenStreetMap](https://osm.org)
+- Radrouten, Wanderwege und Spazierwege
+- Höhenlinien, Höhenprofile, Gipfel und Steigungen
+- Turn-by-Turn-Navigation für Fußgänger, Radfahrer und Autos mit Sprachführung
+- Schnelle Offline-Suche auf der Karte
+- Export und Import von Lesezeichen im KML/KMZ-Format (GPX ist [geplant](https://github.com/organicmaps/organicmaps/issues/624))
+- Dunkler Modus zum Schutz Ihrer Augen
+- Länder und Regionen nehmen nicht viel Platz in Anspruch
+- Kostenlos und Open-Source
+
+## Why Organic?
+
+Organic Maps ist rein und organisch, mit Liebe gemacht:
+
+- Respektiert Ihre Privatsphäre
+- Schont Ihren Akku
+- Keine unerwarteten Gebühren für mobile Daten
+
+Organic Maps ist frei von Trackern und anderen schädlichen Dingen:
+
+- Keine Werbung
+- Kein Tracking
+- Keine Datenerfassung
+- Kein Telefonieren nach Hause
+- Keine lästige Registrierung
+- Keine obligatorischen Tutorials
+- Kein lästiger E-Mail-Spam
+- Keine Push-Benachrichtigungen
+- Keine Crapware
+- ~~Keine Pestizide~~ - rein und organisch!
+
+Der Antrag wird geprüft durch <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a>:
+<br/>
+<img src='/images/privacy/exodus.png' width='400'>
+
+Organic Maps fordert keine übermäßigen Berechtigungen an, um Sie auszuspionieren:
+
+<img src='/images/privacy/om.jpg' width='400'>
+<img src='/images/privacy/mm.jpg' width='400'>
+
+Wir von Organic Maps glauben, dass die Privatsphäre ein grundlegendes Menschenrecht ist:
+
+- Organic Maps ist ein Open-Source-Projekt, das von einer unabhängigen Gemeinschaft betrieben wird.
+- Wir schützen deine Privatsphäre vor den neugierigen Augen von Big Tech
+- Bleiben Sie sicher, egal wo Sie sind
+
+Lehne Überwachung ab - genieße deine Freiheit.
```

Comment:
> It would be possible to use informal language for the whole website to make it a little friendlier.

Please feel free to use informal. There is no need to follow English word-by-word.